docs/how_to/parse_time.rst

```diff
@@ -0,0 +1,122 @@
+------------------
+
+.. code-block:: python
+
+    >>> parse_time(time.gmtime(0))
+    <Time object: scale='utc' format='isot' value=1970-01-01T00:00:00.000>
+
+`datetime.datetime` and `datetime.date`
+----------------------------------------
+
+.. code-block:: python
+
+    >>> parse_time(datetime(1990, 10, 15, 14, 30))
+    <Time object: scale='utc' format='datetime' value=1990-10-15 14:30:00>
+    >>> parse_time(date(2023, 4, 22))
+    <Time object: scale='utc' format='iso' value=2023-04-22 00:00:00.000>
+
+`pandas` time objects
+---------------------
+
+e.g. `pandas.Timestamp`, `pandas.Series` and `pandas.DatetimeIndex`
+
+.. code-block:: python
+
+    >>> parse_time(pandas.Timestamp(datetime(1966, 2, 3)))
+    <Time object: scale='utc' format='datetime64' value=1966-02-03T00:00:00.000000000>
+    >>> parse_time(pandas.Series([[datetime(2012, 1, 1, 0, 0), datetime(2012, 1, 2, 0, 0)],
+    ...                           [datetime(2012, 1, 3, 0, 0), datetime(2012, 1, 4, 0, 0)]]))
+    <Time object: scale='utc' format='datetime' value=[[datetime.datetime(2012, 1, 1, 0, 0) datetime.datetime(2012, 1, 2, 0, 0)]
+                                                       [datetime.datetime(2012, 1, 3, 0, 0) datetime.datetime(2012, 1, 4, 0, 0)]]>
+    >>> parse_time(pandas.DatetimeIndex([datetime(2012, 1, 1, 0, 0),
+    ...                                  datetime(2012, 1, 2, 0, 0),
+    ...                                  datetime(2012, 1, 3, 0, 0),
+    ...                                  datetime(2012, 1, 4, 0, 0)]))
+    <Time object: scale='utc' format='datetime' value=[datetime.datetime(2012, 1, 1, 0, 0)
+                                                       datetime.datetime(2012, 1, 2, 0, 0)
+                                                       datetime.datetime(2012, 1, 3, 0, 0)
+                                                       datetime.datetime(2012, 1, 4, 0, 0)]>
+
+`numpy.datetime64`
+------------------
+
+.. code-block:: python
+
+    >>> parse_time(np.datetime64('2014-02-07T16:47:51.008288123'))
+    <Time object: scale='utc' format='isot' value=2014-02-07T16:47:51.008>
+    >>> parse_time(np.array(['2014-02-07T16:47:51.008288123', '2014-02-07T18:47:51.008288123'],
+    ...                     dtype='datetime64'))
+    <Time object: scale='utc' format='isot' value=['2014-02-07T16:47:51.008' '2014-02-07T18:47:51.008']>
+
+Formats handled by `astropy.time.Time`
+--------------------------------------
+
+See this list of all allowed formats, see `this list <https://docs.astropy.org/en/stable/time/#time-format>`__
+
+.. code-block:: python
+
+    >>> parse_time(1234.0, format='jd')
+    <Time object: scale='utc' format='jd' value=1234.0>
+    >>> parse_time('B1950.0', format='byear_str')
+    <Time object: scale='tt' format='byear_str' value=B1950.000>
+
+``anytim`` output
+------------------
+
+Format output by the ``anytim`` routine in SSW (see the documentation for `~sunpy.time.TimeUTime` for more information):
```

Comment:
```suggestion
Format output by the ``anytim`` routine in SolarSoft (see the documentation for `~sunpy.time.TimeUTime` for more information):
```
To avoid acroyms?